src/MapViewController.swift

```diff
@@ -487,6 +495,34 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
     tgViewController.markerRemove(marker)
     return
   }
+
+  //MARK: - private
+
+  private func setupTgControllerView() {
+    tgViewController.view.frame = CGRect(x: 0, y: 0, width: self.view.bounds.width, height: self.view.bounds.height-tabBarHeight)
```

Comment:
The view's height doesn't adjust for the tab bar so I do need to adjust for it. I've updated setting of the `tgViewController` to use constraints.